adcc/backends/psi4.py

```diff
@@ -64,6 +64,15 @@ def pe_induction_elec_ao(dm):
                 )[1]
             return pe_induction_elec_ao
 
+    @property
+    def pcm_potential_elec(self):
+        if self.wfn.PCM_enabled():
+            def pcm_potential_elec_ao(dm):
+                return psi4.core.PCM.compute_V(
+                    self.wfn.get_PCM(),
+                    psi4.core.Matrix.from_array(dm.to_ndarray()))
```

Comment:
Can you move the last `)` to the next line?
```suggestion
                return psi4.core.PCM.compute_V(
                    self.wfn.get_PCM(),
                    psi4.core.Matrix.from_array(dm.to_ndarray())
                )
```